requirements.txt

```diff
@@ -8,7 +8,6 @@ redis==2.10.5
 requests==2.20.0
 plyvel==0.9
 netaddr==0.7.18
-antlr4-python2-runtime==4.5.2
```

Comment:
Any reason this was deleted? and requirements-web merged into requirements?

Reply:
antlr4-python2-runtime is an indirect requirement of stix2-patterns.  this change is just to avoid updating both versions together since stix2-patterns requires a specific version.

I will undo the merge of the requirements-web requirements.  PyCharm doesn't know about multiple requirements file -- accidentally checked this in.